packages/salesforcedx-vscode-core/test/vscode-integration/conflict/metadataCacheService.test.ts

```diff
@@ -92,6 +93,7 @@ describe('Metadata Cache', () => {
       pollStatusStub.callsFake(() => {});
       operationStub.resolves(mockOperation);
       processStub.resolves(undefined);
+      sinon.stub(ConfigUtil, 'getUserConfiguredApiVersion').resolves('55.0');
```

Comment:
Without this, this test failed - complaining about a usage of process.chdir()
<img width="1000" alt="2022-08-18_06-32-03" src="https://user-images.githubusercontent.com/46458081/185459679-d2caf4f9-9f00-4707-9714-5ed8e65955e9.png">

Reply:
good ol uningration tests